api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/MultiValueEditSubmission.java

```diff
@@ -303,7 +303,10 @@ public void addLiteralToForm(EditConfigurationVTwo editConfig, FieldVTwo field,
 					String rangeLang = field.getRangeLang();  //UQAM  Default value
 					try {
 						if (_vreq != null ) {
-							rangeLang	= _vreq.getLocale().getLanguage() + "-"+_vreq.getLocale().getCountry();
+                            rangeLang	= _vreq.getLocale().getLanguage();
```

Comment:
The formatting in the VIVO codebase is... inconsistent. However, it is helpful not to exacerbate the chaos. Would you mind changing the indentation of the lines you added to either use spaces or tabs in order to match the lines around it?

Reply:
understandable, i changed it.